.github/workflows/release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup kustomize
+        uses: imranismail/setup-kustomize@v2
+
+      - name: Generate release manifests
+        run: |
+          make manifests
+          kustomize build ./manifests > ./manifests/vm-console-proxy.yaml
+
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Upload generated release manifests
```

Comment:
@akrejcir Also, from what I understand, step `Upload generated release manifests` will add generated manifest to the release artifacts, e.g. [1]. It's needed here as well?

[1] https://github.com/kubevirt/kubevirt-tekton-tasks/releases/tag/v0.13.0

Reply:
Yes, a release consists of:
- An image uploaded to quay like: `quay.io/kubevirt/vm-console-proxy:v0.1.0`
- Generated manifests that are part of the Github release. They can be used to easily install the component without cloning the repo.

Reply:
Got it, thanks!